tests/sentry/integrations/github/test_integration.py

```diff
@@ -547,21 +558,21 @@ def test_github_prevent_install_until_pending_deletion_is_complete(self):
         ).exists()
 
     @responses.activate
-    def test_get_tree_for_repo(self):
+    def test_get_trees_for_org(self):
```

Comment:
Eventually this function will be the one testing the derivation of code mappings.